tests/functional/t/typevar_name_missing_variance.py

```diff
@@ -0,0 +1,41 @@
+"""Test case for typevar-name-incorrect-variance."""
+
+from typing import TypeVar
+
+# Type variables without variance
+T = TypeVar("T")
+T_co = TypeVar("T_co")  # [typevar-name-incorrect-variance]
+T_contra = TypeVar("T_contra")  # [typevar-name-incorrect-variance]
+T_scores_contra = TypeVar("T_scores_contra")  # [typevar-name-incorrect-variance]
+
+# Type variables not starting with T
+N = TypeVar("N")
+N_co = TypeVar("N_co", covariant=True)
+N_contra = TypeVar("N_contra", contravariant=True)
+
+# Tests for combinations with contravariance
+CT_co = TypeVar("CT_co", contravariant=True)  # [typevar-name-incorrect-variance]
+CT_contra = TypeVar("CT_contra")  # [typevar-name-incorrect-variance]
+CT_contra = TypeVar("CT_contra", contravariant=True)
+
+# Tests for combinations with covariance
+VT = TypeVar("VT", covariant=True)  # [typevar-name-incorrect-variance]
+VT_contra = TypeVar("VT_contra", covariant=True)  # [typevar-name-incorrect-variance]
+VT_co = TypeVar("VT_co", covariant=True)
+
+# Tests for combinations with bound
+VT = TypeVar("VT", bound=True)
+VT_co = TypeVar("VT_co", bound=True)  # [typevar-name-incorrect-variance]
+VT_contra = TypeVar("VT_contra", bound=True)  # [typevar-name-incorrect-variance]
```

Comment:
```suggestion
VT = TypeVar("VT", bound=int)
VT_co = TypeVar("VT_co", bound=int)  # [typevar-name-incorrect-variance]
VT_contra = TypeVar("VT_contra", bound=int)  # [typevar-name-incorrect-variance]
```
`True` is not a valid type